### PR TITLE
feat(VirtualizedList): Adds support for badges

### PR DESCRIPTION
**Pull request by talend-glorieux**
**What is the problem this PR is trying to solve?**
In order to display specific information on the list we need to be able to render Badges components.

**What is the chosen solution to this problem?**
Adds support for Badges component in Virtualized lists.

**Please check if the PR fulfills these requirements**
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

### CODE REVIEW

File: packages/components/.storybook/preview-head.html [hunk 1]
```diff
@@ -28,20 +29,29 @@
...
     .virtualized-list div.tc-list-cell-author {
         flex: 1 0 90px;
     }
+
     .virtualized-list div.tc-list-cell-created,
     .virtualized-list div.tc-list-cell-modified {
         flex: 0 0 90px;
     }
-</style>
+</style>
```
Comment (acateland) [inline packages/components/.storybook/preview-head.html]: missing EOL

File: packages/components/src/Badge/Badge.component.js [hunk 2]
```diff
@@ -28,12 +28,25 @@ function renderDeleteIcon(onClick, id, disabled, t) {
...
+	onSelect,
+	selected,
+	disabled,
+	t,
+	style,
+	className,
+}) {
 	const containerClasses = classNames(
-		'tc-badge', theme['tc-badge'],
+		'tc-badge',
```
Comment (acateland) [inline packages/components/src/Badge/Badge.component.js]: there is not value tested to add the tc-badge global and local class is this wanted ?

File: packages/components/src/VirtualizedList/CellBadge/CellBadge.component.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Badge from '../../Badge';
+
+/**
+ * Cell renderer that displays a badge
+ */
+function CellBadge({ cellData, rowIndex }) {
+	return <Badge id={`${rowIndex}`} label={cellData} selected />;
```
Comment (acateland) [inline packages/components/src/VirtualizedList/CellBadge/CellBadge.component.js]: shouldn't id be prefixed ? tc-list-badge-id or something like that

Comment (jsomsanith) [inline packages/components/src/VirtualizedList/CellBadge/CellBadge.component.js]: hard coded-selected : is it for the style ?

File: packages/components/src/VirtualizedList/CellBadge/CellBadge.test.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+import CellBadge from './CellBadge.component';
+
+describe('CellActions', () => {
+	it('should render', () => {
+		// when
+		const wrapper = shallow(<CellBadge cellData={{ label: 'streaming' }} rowIndex={25} />);
+
+		// then
+		expect(wrapper.node).toMatchSnapshot();
```
Comment (jmfrancois) [inline packages/components/src/VirtualizedList/CellBadge/CellBadge.test.js]: getNode please

File: packages/components/src/VirtualizedList/CellBadge/CellBadge.component.js (hunk 3 above)
Comment (jsomsanith) [reply packages/components/src/VirtualizedList/CellBadge/CellBadge.component.js]: The badge component add a prefix, passing the line nb outputs `tc-badge-select-4`

Comment (talend-glorieux) [reply packages/components/src/VirtualizedList/CellBadge/CellBadge.component.js]: Yep. I know. It should be retrieved from the columnData. :)

File: packages/components/src/Badge/Badge.component.js (hunk 2 above)
Comment (talend-glorieux) [reply packages/components/src/Badge/Badge.component.js]: That's why I've added the className to be able to specify it from the outside.

**Pull request merged**